app/controllers/api_new/payouts_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
```

Comment:
Between the three of these files the License is slightly different -- I assume it's all good, though.

Reply:
Good catch.

The license header and name were slightly changed in upstream Houdini. I must have copied something in from upstream and tossed this here. The licenses are identical in terms though so they're interchangeable

We should probably standardize on the upstream header but I've never got around to doing so.